.sauce/cypress.yml

```diff
@@ -13,6 +13,8 @@ suites:
     match: ".*.(spec|test).js$"
     capabilities:
```

Comment:
should we remove this then?

Reply:
I actually updated that. It still remains backwards compatibility for `capabilities` in `suites`. I removed `capabilities` under `project` actually.